server/migrations/20200708183925-subscription-model-upgrade.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable quotes */
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+  // add columns
+    await queryInterface.addColumn('Subscriptions', 'chain_id', { type: DataTypes.STRING, allowNull: true });
+    await queryInterface.addColumn('Subscriptions', 'community_id', { type: DataTypes.STRING, allowNull: true });
+    await queryInterface.addColumn('Subscriptions', 'offchain_thread_id', { type: DataTypes.INTEGER, allowNull: true });
+    await queryInterface.addColumn('Subscriptions', 'offchain_comment_id', { type: DataTypes.INTEGER, allowNull: true });
+    await queryInterface.addColumn('Subscriptions', 'chain_event_type_id', { type: DataTypes.STRING, allowNull: true });
+    await queryInterface.addColumn('Subscriptions', 'chain_entity_id', { type: DataTypes.STRING, allowNull: true });
```

Comment:
Do we ever use this `chain_entity_id` field?

Reply:
Not in the migration, but we should be in the case of new subscription on a `chainEntity`, unless I misunderstood what they're for.